src/Components/Playground/Playground.tsx

```diff
@@ -44,9 +44,9 @@ export const Playground = () => {
   };
 
   useEffect(() => {
-    window.onkeydown = handleKeyPress;
+    window.addEventListener('keydown', handleKeyPress);
     return () => {
-      window.onkeydown = null;
+      window.removeEventListener('keydown', handleKeyPress);
     };
   });
```

Comment:
Not entirely correct. This code will remove the previous and add a new event listener with each render because the hook is executed with each render (has not dependency array)